suites/message/message_application.go

```diff
@@ -46,13 +46,26 @@ func MessageTest_MessageApplicationEdgecases(t *testing.T, factory state.Factori
 			td.MessageProducer.Transfer(alice, alice, chain.Value(transferAmnt), chain.Nonce(0), chain.GasPrice(10), chain.GasLimit(1)),
 			exitcode.SysErrOutOfGas)
 
-		// Expect Message application to fail due to lack of gas when sender address is unknown
+		// Expect Message application to fail due to lack of gas when the message being sent has invalid destination.
```

Comment:
I think the original comment is correct. I find the argument order of `Transfer` to be very confusing though so had to go inspect it to confirm this.

Reply:
Filed https://github.com/filecoin-project/chain-validation/issues/197 to address parameter ordering.